storage/src/context.rs

```diff
@@ -290,6 +294,21 @@ impl ContextApi for TezedgeContext {
         let merkle = self.merkle.read().expect("lock poisoning");
         merkle.get_staged_root_hash()
     }
+
+    fn block_applied(&self) -> Result<(), ContextError> {
+        let mut merkle = self.merkle.write().expect("lock poisoning");
```

Comment:
@mat13mn 
please change expect to error handling,

anyway, do we need to do anything here?
if we do not support gc here now, I would just leaeve it as `Ok(())` to avoid unnecessecery locking

Reply:
we support GC but its implementation detail of KV backend passed to merkle storage so at this point we dont know if its supported or not